SQL/0000-00-00-schema.sql

```diff
@@ -724,6 +732,38 @@ INSERT INTO mri_protocol (Center_name,Scan_type,TR_range,TE_range,time_range) VA
   ('ZZZZ',44,'2000-2500','2-5',NULL),
   ('ZZZZ',45,'3000-9000','100-550',NULL);
 
+CREATE TABLE `mri_protocol_group` (
+    `MriProtocolGroupID`   INT(2) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `Name`                 VARCHAR(255)    NOT NULL,
+    PRIMARY KEY (`MriProtocolGroupID`)
+) ENGINE = InnoDB  DEFAULT CHARSET=utf8;
+
+INSERT INTO `mri_protocol_group` VALUES(1, "Default MRI protocol group");
+
+CREATE TABLE `mri_protocol_group_rel` (
+    `MriProtocolGroupID` INT(2) UNSIGNED  NOT NULL,
+    `MriProtocolID`      INT(11) UNSIGNED NOT NULL,
+    CONSTRAINT `FK_mri_protocol_group_rel_1` FOREIGN KEY (`MriProtocolGroupID`) REFERENCES `mri_protocol_group` (`MriProtocolGroupID`),
+    CONSTRAINT `FK_mri_protocol_group_rel_2` FOREIGN KEY (`MriProtocolID`)      REFERENCES `mri_protocol` (`ID`),
+    CONSTRAINT `UK_mri_protocol_group_rel`   UNIQUE (`MriProtocolID`)
```

Comment:
If MriProtocolID is unique, then the MriProtocolGroupID should be added to the mri_protocol table.

Reply:
Ok. I'll remove table `mri_protocol_group_rel` then.